app/src/main/res/values-hr/strings.xml

```diff
@@ -107,79 +107,79 @@
+    <string name="faq_6_text">Može se dogoditi da vidite neke neobične albume. Možete ih jednostavno izuzeti tako da ih dugo pritisnete i odaberete Izuzmi. U sljedećem dijaloškom okviru možete odabrati glavnu mapu, čime će te spriječiti prikazivanje ostalih povezanih albuma.</string>
+    <string name="faq_7_title">Mapa s fotografijama se ne prikazuje, što mogu učiniti?</string>
+    <string name="faq_7_text">To može imati više razloga, ali rješenje je jednostavno. Samo idite u Postavke -> Upravljanje uključenim mapama, odaberite ikonu Plus i prijeđite na traženu mapu.</string>
+    <string name="faq_8_title">Što ako želim vidjeti samo nekoliko određenih mapa?</string>
+    <string name="faq_8_text">Dodavanje direktorija u uključene mape ne izuzima ništa automatski. Da biste to učinili, idite na "Postavke", a zatim "Upravljanje izuzetim mapama", izuzmite korijenski direktorij \"/\", a zatim dodajte željene direktorije u "Postavke" i "Upravljanje uključenim mapama".
+        Samo će odabrane mape biti vidljive, jer su izuzimanja i uključivanja rekurzivni, a ako je mapa isključena i uključena, bit će prikazana.</string>
+    <string name="faq_9_title">Slike na cijelom zaslonu imaju čudne artefakte, mogu li nekako poboljšati kvalitetu?</string>
+    <string name="faq_9_text">Da, u "Postavkama" postoji opcija "Zamjena duboko zumiranih slika s kvalitetnijim slikama", ali slike će biti zamućene ako zumirate previše.</string>
+    <string name="faq_10_title">Mogu li izrezati slike pomoću ove aplikacije?</string>
+    <string name="faq_10_text">Da, možete obrezati slike u uređivaču povlačenjem uglova. Možete doći do uređivača dugim pritiskom na minijaturu slike i odabirom Uređivanje ili odabirom Uredi iz prikaza preko cijelog zaslona.</string>
 
     <!-- Strings displayed only on Google Playstore. Optional, but good to have -->
     <!-- Short description has to have less than 80 chars -->
-    <string name="app_short_description">Galerija za pregledavanje slika, GIFova i videa bez reklama.</string>
+    <string name="app_short_description">Galerija za gledanje fotografija i videozapisa bez oglasa.</string>
     <string name="app_long_description">
         Jednostavan alat za pregled slika, GIFova i videa. Datoteke možete sortirati po datumu, veličini, imenu i to uzlazno i silazno. Također možete zumirati slike. Medijski sadržaj se prikazuje u višestrukim stupcima ovisno o veličini ekrana, a vi samo možete birati broj stupaca s gestom štipkanja. Možete preimenovati, dijeliti, brisati, kopirati, premještati datoteke. Slike također možete izrezati, rotirati ili postaviki kao pozadinu ekrana, odmah iz aplikacije.
 
         Galerija se također može koristiti za pregledavanje slika i videa u drugim aplikacijama, prikačivanja datoteka u e-mail aplikacije itd. Savršeno za svakodnevno korištenje.
 
         The fingerprint permission is needed for locking either hidden item visibility, or the whole app.
```

Comment:
how about this?

Reply:
Sorry on missing that one, going to create a correction right now.